### DISCUSSION TITLE
Add `repository` and `owner` on yanked crate page

### DISCUSSION THREAD

**Discussion by bestia-dev** [rust-lang/crates.io#4322] [Feature Requests]
On a normal crate page on crates.io, there are `repository` and `owner` informations.
But on an "all versions yanked" crate page I cannot find these important information.
Example [lmake_cargo_toml_to_md](https://crates.io/crates/lmake_cargo_toml_to_md) \:

![image](https://user-images.githubusercontent.com/31509965/129444557-803aa422-2baa-4ca5-b1d1-94e8498895e8.png)

Sure I can find the `owner` if I go to `all versions`.
And I can find the `repository` in the Search results: 

![image](https://user-images.githubusercontent.com/31509965/129445003-eb2ab2ac-95b7-4cd9-8a06-c56826d69db0.png)

But It would be nice to have these important pieces of information on the "all yanked" page.

**Comment by Turbo87** [comment]
FWIW this appears to be a duplicate of https://github.com/rust-lang/crates.io/issues/502
